docs/api-reference/deck.md

```diff
@@ -297,6 +297,12 @@ deck.setProps({...});
 
 See the Properties section on this page for more detail on which props can be set.
 
+##### `redraw`
+
+`deck.redraw()`
+
+Force deck to redraw (if necessary) immediately. By default, deck flushes all changes to the canvas on each animation frame. This behavior might cause the deck canvas to fall out of sync with other components if synchronous updates are required. Only use this method if the render cycle should be managed manually.
```

Comment:
"Force" is a little confusing, since it still does all the checks. Maybe something like "Attempt to draw immediately, rather than waiting for the next draw cycle" would be clearer. Should also mention that this isn't recommended outside an rAF callback.